dependent/wissensspeicher.csl

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="de-DE">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Wissensspeicher</title>
+    <title-short>ws</title-short>
+    <id>http://www.zotero.org/styles/wissensspeicher</id>
+    <link rel="self" href="http://www.zotero.org/styles/wissensspeicher"/>
+    <link href="http://www.zotero.org/styles/thieme-german" rel="independent-parent"/>
```

Comment:
I think you are misunderstanding what a dependent style is? You wouldn't have any actual style code. Just the info section. If anything in the style is different from Thieme, it's not dependent

Reply:
I haven't found a style that even comes close to fitting. That's why I tried to build one from scratch. But that doesn't seem to work. Is there perhaps a manual in German for this somewhere?

Reply:
Es gibt ein deutsches YouTube Video von einem Webinar das ich an der Uni Mannheim gemacht habe.
Aber das Problem hier ist nicht der Stil an sich, sondern dass Du ihn als "dependent" behandelst, obwohl er das nicht ist. Der Stil sollte nicht im /dependent Ordner sein und auch keinen Independent-parent Link haben

Reply:
OK, Danke! Ich hoffe ich habe es jetzt verstanden.